test/NuGet.Core.FuncTests/Dotnet.Integration.Test/MsbuildIntegrationTestFixture.cs

```diff
@@ -234,6 +245,48 @@ internal void BuildProject(string workingDirectory, string projectName, string a
             Assert.True(result.Item3 == "", $"Build failed with following message in error stream :\n {result.AllOutput}");
         }
 
+        internal TestDirectory CreateTestDirectory()
+        {
+            var testDirectory = TestDirectory.Create();
+
+            WriteGlobalJson(testDirectory);
+
+            return testDirectory;
+        }
+
+        internal SimpleTestPathContext CreateSimpleTestPathContext()
+        {
+            var simpleTestPathContext = new SimpleTestPathContext();
+
+            WriteGlobalJson(simpleTestPathContext.WorkingDirectory);
+
+            var addSourceArgs = new AddSourceArgs()
```

Comment:
SimpleTestPathContext should be creating a nuget.config already.

Reply:
It does. This code adds the template cache's global package folder as a package source to that nuget.config that SimpleTestPathContext created.

Reply:
added a comment.